src/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { EmojiPicker } from './EmojiPicker'
 import { EmojiKeyboard } from './EmojiKeyboard'
+export { DeleteButton } from './components/DeleteButton'
```

Comment:
Is this new export done on purpose? It seems to have nothing directly in common with the fix, but if for good reasons, no big deal.